circleci/Dockerfile

```diff
@@ -0,0 +1,77 @@
+# We cannot use Ubuntu 22.04 because the E2E tests
+# are currently using a Docker Compose v1 imaged based on Debian.
+# The glibc version is too old to allow running CGO binaries built on Ubuntu 22.04
+# We'll be able to migrate when we get rid of Docker Compose or use Docker Compose v2
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Pre-requisites
+# Python 3 dev is required for rtloader
+RUN set -ex \
+    && apt-get update && apt-get upgrade -y \
+    && apt-get install -y --no-install-recommends \
+        curl \
+        default-jre \
+        doxygen \
+        file \
+        g++ \
+        gcc \
+        git \
+        gnupg ca-certificates \
+        graphviz \
+        libpq-dev \
+        libsnmp-base \
+        libsnmp-dev \
+        libssl-dev \
+        libsystemd-dev \
+        make \
+        pkg-config \
+        python3 \
+        python3-dev \
+        python3-distutils \
+        python3-pip \
+        python3-setuptools \
+        python3-yaml \
+        snmp-mibs-downloader \
+        ssh \
+        xz-utils
+
+# Golang
+ENV GO_VERSION 1.19.9
```

Comment:
Took the opportunity to remove usage of gimme for coherence with this [PR](https://github.com/DataDog/datadog-agent-buildimages/pull/370)